nexus/db-macros/Cargo.toml

```
@@ -18,4 +18,5 @@ syn = { workspace = true, features = ["extra-traits"] }
 omicron-workspace-hack.workspace = true
 
 [dev-dependencies]
+expectorate.workspace = true
 prettyplease.workspace = true
```

```suggestion
prettyplease = { workspace = true, features = ["verbatim"] }
```

My guess is this will resolve the divergence. I will leave it to you to look into how come the feature can be disabled on @sunshowers's machine and enabled in the CI machine.

---

Hah, thanks. Yeah, this looks like a feature unification thing.

---

@dtolnay if we say "serde" into a mirror three times, do you appear?